tests/components/history/test_init.py

```diff
@@ -130,13 +130,16 @@ def test_get_significant_states_with_initial(hass_history) -> None:
     """
     hass = hass_history
     zero, four, states = record_states(hass)
-    one = zero + timedelta(seconds=1)
     one_and_half = zero + timedelta(seconds=1.5)
     for entity_id in states:
         if entity_id == "media_player.test":
             states[entity_id] = states[entity_id][1:]
         for state in states[entity_id]:
-            if state.last_changed == one:
+            # If the state is recorded before the start time
```

Comment:
This was a bug. Only states before the start of the query should ever have the start time state